# Flight Control Advanced
uint8 control_flag
float32 x
float32 y
float32 z
float32 yaw
float32 x_feedforward
float32 y_feedforward

